docs/production/upload-backends.md

```diff
@@ -35,20 +35,23 @@ as world-readable, whereas the "uploaded files" one is not.
 
 1. Comment out the `LOCAL_UPLOADS_DIR` setting in
    `/etc/zulip/settings.py` (add a `#` at the start of the line).
+   And uncomment the `S3_ENDPOINT_URL` setting.
```

Comment:
Saying "uncomment" implies that you won't need to modify it, which doesn't seem accurate -- the template sets it to `None`, the same as the default.  Related to that, it would be good if folks using the standard S3 endpoint didn't have to modify this in any way.  Can we have the value in `default_settings` be whatever is necessary to upload to S3, so that we don't introduce another step here for folks just using S3 itself?  The end of the next step (below) can call out modifying the `S3_ENDPOINT_URL` if folks are using something S3-compat that isn't S3 itself.